Arrt/View/ModelEditor/Stats/StatsGraph.cpp

```diff
@@ -55,7 +55,20 @@ namespace
 
     QString toString(double d)
     {
-        return DoubleFormatter::toString(d, "%.2f", true);
+        // Print large numbers in a humand-friendly way. We add a space
+        // at the end so numbers with units don't look weird, e.g. "1M ms"
+        QString suffix;
+        if (d > 1000000)
```

Comment:
You can also just check it's divisible by 1000000 (or 1000), which should be the case for how the graph is computed

Reply:
I've added two functions and made it clearer what it's used for.